### DISCUSSION TITLE
Missing plugins

### DISCUSSION THREAD

**Discussion by fojerhar** [apohl79/audiogridder#964] [General]
HI

Would it be possible to solve, if some plugins are missing from the computer the missing slot would be in bypass not in mute?
Sometimes I have to continue mixing a project with AG slot is at another studio, where I do not have all my plugins installed.
If one plug is missing from the chain, the whole chain is in mute, if I delete the missing ones, sound comes back, but in this case I cannot continue mixing at the main computer, because the plugs were deleted.
thank you
btw, beta9 works really well with protools on M1ultra..... :)

**Comment by apohl79** [comment]
will create a feature request from this. 👍 cheers

**Reply by fojerhar** [reply]
amazing, thank you
